images/installer/root/etc/inventory-generator-config.yaml

```diff
@@ -0,0 +1,23 @@
+---
+# meta config
+master_config_path: null  # defaults to /opt/app-root/src/master-config.yaml
+admin_kubeconfig_path: null  # defaults to /opt/app-root/src/.kube/config
+
+# default user configuration
+ansible_ssh_user: ec2-user
+ansible_become: "yes"
+ansible_become_user: "root"
+
+# openshift-ansible inventory vars
+openshift_uninstall_images: false
+openshift_install_examples: true
+openshift_deployment_type: origin
+
+openshift_release: 3.6
+openshift_image_tag: v3.6.0
+openshift_hosted_logging_deploy: null  # defaults to "true" if loggingPublicURL is set in master-config.yaml
```

Comment:
I'm thinking with this and `admin_kubeconfig_path` above, I could at least default them to their default values instead of `null`. I think having these present in the default config file gives a better glance of what can be tweaked